vllm/model_executor/models/prithvi_geospatial_mae.py

```diff
@@ -173,6 +188,13 @@ def _parse_and_validate_multimodal_data(
 
         return pixel_values, location_coords
 
+    def get_input_embeddings(self, input_ids: torch.Tensor) -> torch.Tensor:
+        # We do not really use any input tokens and therefore no embeddings
+        # to be calculated. However, due to the mandatory token ids in
+        # the input prompt we pass one token and the size of the dummy
+        #  embedding tensors must reflect that.
+        return torch.empty(input_ids.shape)
+
```

Comment:
I don't see the `get_multimodal_embeddings()` function here. How is this not failing in `GPUModelRunner.profile_run()`?

Reply:
Yes, but if we're adding a new programming model for MM inputs I think it would be nice to handle this. If not a full solution, perhaps a test to disable cuda graphs and memory profiling for this special case.

But I'm thinking that maybe the raw MM inputs will be hard to generalize. Perhaps a hacky solution for this particular model would be to store the raw MM inputs in the `get_multimodal_embeddings()` method and then use them when `forward()` is called.